main.go

```diff
@@ -30,14 +30,15 @@ func makeFunctionURL(req *queue.Request, config *QueueWorkerConfig, path, queryS
 	if len(path) > 0 {
 		pathVal = path
 	}
-	functionURL := fmt.Sprintf("http://%s%s:8080%s%s",
+	functionURL := fmt.Sprintf("http://%s%s:%d%s%s",
```

Comment:
This has been changed incorrectly.

This line does not invoke via the gateway.

Reply:
@acornies FYI ^

Reply:
Changed.